charts/index.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+entries:
+  secrets-store-csi-driver:
+  - apiVersion: v1
+    appVersion: 0.0.9
+    created: "2020-04-28T00:44:38.442052+01:00"
+    description: A Helm chart to install the SecretsStore CSI Driver inside a Kubernetes
+      cluster.
```

Comment:
nit: move cluster to previous line

Reply:
This file is generated by `helm repo index` and will constantly be generated with cluster on  a separate line

Reply:
got it!